Docs: small typos and grammar fixes (#3543)
ethereum/web3.js: docs/web3-eth-contract.rst

    web3.eth.Contract.handleRevert
    contract.handleRevert // on contract instance

The ``handleRevert`` options property defaults to ``false`` and returns the revert reason string if enabled on :ref:`send <contract-send>` or :ref:`call <contract-call>` of a contract method.

> .. note:: The revert reason string and signature exists as property on the returned error.

Reviewer: ```suggestion
.. note:: The revert reason string and signature are properties on the returned error.
```
